.github/workflows/ci_test-tpu.yml

```diff
@@ -3,10 +3,8 @@ name: TPU tests
 on:
   push:
     branches: [master, "release/*"]
-# TODO: temporal disable TPU testing until we find way how to pass credentials to forked PRs
-#  pull_request:
-#    branches:
-#      - master
+  pull_request:
+    branches: [master, "release/*"]
```

Comment:
this won't work for forkerd PRs

Reply:
we didn't know it has to be changed in the GitHub settings